src/llm-eval/README.md

```diff
@@ -0,0 +1,297 @@
+# Evaluating a LLM with .NET
+
+This repo contains a library and samples for evaluating the output of an LLM, similar to what we  uses in a RAG architecture. There are many parameters that affect the quality and style of answers generated by a chat app, such as the system prompt, search parameters, and GPT model parameters.
+
+This repository offers tools and samples to make it easier to run evaluations on Large Language Models.
```

Comment:
Is this a duplicate? If so, I'd recommend going with the first version.

Reply:
Yes, thanks. Keeping the 1st one.